debug/test-opencv-capture.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
+
+# To see CV debug information, do
+#
+#    OPENCV_VIDEOIO_DEBUG=1 debug/test-camera.py
```

Comment:
This header comment should probably be a module docstring and be updated to the new name of the .py file.  

I don't know if the constant `OPENCV_VIDEOIO_DEBUG` really works?